src/components/Contact.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Contact.css';
+import InLineSpinner from './InLineSpinner';
+import Avatar from './Avatar';
+import Button from './Button';
+import Checkbox from './Checkbox';
+
+class Contact extends React.Component {
+  constructor(props) {
+    super(props);
+    const { contact } = this.props;
+    this.state = { contact };
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  get createFavouriteContactJSX() {
+    return (
```

Comment:
you can return directly the jsx div since is only one single line:
```
 get createFavouriteContactJSX() {
    return <div>getFavouriteContactJSX</div>;
}
```